src/test/java/catalinaTema5/DemoqaElements/RadioButtons.java

```diff
@@ -0,0 +1,22 @@
+package catalinaTema5.DemoqaElements;
+
+import catalinaTema5.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class RadioButtons {
+    public static void main(String[] args){
+        goToRadioButtonsPage();
+ }
+
+    public static  void goToRadioButtonsPage() {
+     ChromeDriver driver = BrowserManager.createChromeDriver();
+     driver.get("https://demoqa.com/radio-button");
+        WebElement radioButtonNo = driver.findElement(By.xpath("//label[@class='custom-control-label disable' and @for='noRadio']"));
```

Comment:
Here it should be disabled and not disable in the name of the class. Please make the necessary adjustment. Thanks!